lnbits/core/helpers.py

```diff
@@ -0,0 +1,42 @@
+import importlib
+import re
+
+from loguru import logger
+
+from lnbits.extension_manager import Extension
+
+from . import db as core_db
+from .crud import update_migration_version
+
+
+async def migrate_extension_database(ext: Extension, current_version):
+    try:
+        ext_migrations = importlib.import_module(f"{ext.module_name}.migrations")
+        ext_db = importlib.import_module(ext.module_name).db
+    except ImportError as e:
+        logger.error(e)
+        raise ImportError(
+            f"Please make sure that the extension `{ext.code}` has a migrations file."
+        )
+
+    async with ext_db.connect() as ext_conn:
+        await run_migration(ext_conn, ext_migrations, current_version)
+
+
+async def run_migration(db, migrations_module, current_version):
+    matcher = re.compile(r"^m(\d\d\d)_")
+    db_name = migrations_module.__name__.split(".")[-2]
+    for key, migrate in migrations_module.__dict__.items():
+        match = match = matcher.match(key)
```

Comment:
is this an error? ```match = match = matcher.match(key)```too much matching for me :)

Reply:
yeah, looks like it
this code has just been moved here, the same issue is on `main`

Reply:
lol this works? didn't know

Reply:
fixed!
It works was there from the start of time.